app/controllers/dashboards_controller.rb

```diff
@@ -5,6 +5,6 @@ def show
     @showing_all = params[:all_players].present?
     @game = current_user.build_game
     @leaderboard_users = User.by_elo
-    @leaderboard_users = @leaderboard_users.limit(10) unless @showing_all
+    @leaderboard_users = @leaderboard_users.rankable unless @showing_all
```

Comment:
Can we keep the limit also? Otherwise this will show everyone rankable instead of just a top 10